src/snapred/backend/recipe/algorithm/DetectorPeakPredictor.py

```diff
@@ -80,7 +80,7 @@ def PyExec(self) -> None:
             for d in dList:
                 # beta terms
                 beta_T = beta_0 + beta_1 / d**4  # GSAS-I beta
-                beta_d = self.BETA_D_COEFFICIENT * L * np.sin(tTheta / 2) * beta_T  # converted to d-space
+                beta_d = self.BETA_D_COEFFICIENT * L * (np.sin(tTheta / 2) ** -1) * beta_T  # converted to d-space
```

Comment:
why using `** -1` instead of `/`?

Reply:
That is, why $L \times (\sin(\theta/2))^{-1} \times \beta_T$  instead of $L \times \beta_T / \sin(\theta/2)$